msbuild/Xamarin.iOS.Tasks.Core/Xamarin.WatchOS.AppExtension.Common.targets

```diff
@@ -18,6 +18,7 @@ Copyright (C) 2015-2016 Xamarin. All rights reserved.
 	<PropertyGroup>
 		<IsAppExtension>True</IsAppExtension>
 		<IsWatchExtension>True</IsWatchExtension>
+		<IOSDebugOverWiFi>True</IOSDebugOverWiFi>
```

Comment:
the extension runs on the phone so the normal (USB) debugging should be fine right ?

Reply:
Are you sure? I thought the extensions ran on the Watch now.

WatchOS1 extensions ran on the iPhone, but I thought starting with 2.0, they ran on the watch.

Reply:
you're right, the app is a shell